CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ _All notable changes to this project will be documented in this file. This proje
 
 <!-- INSERT HERE -->
 
+## [0.13.0](https://github.com/uber/nebula.gl/compare/v0.12.2...v0.13.0) - TODO
```

Comment:
Just remove this title the tool will auto-add it

Reply:
Yep, just put it there to have details of the breaking change. Will correct when running the changelog script.